ocaml/idl/datamodel.ml

```diff
@@ -1052,7 +1052,8 @@ let _ =
     ~doc:"You attempted to start a VM that's attached to more than one VDI with a timeoffset marked as reset-on-boot." ();
   error Api_errors.vms_failed_to_cooperate [ ]
     ~doc:"The given VMs failed to release memory when instructed to do so" ();
-
+  error Api_errors.ballooning_timeout_before_migration [ "vm" ]
+    ~doc:"Timeout trying to balloon memory before VM migration. If the error persists, try to increase the memory-dynamic-min value." ();
```

Comment:
What? "Try to increase..." There's no doubt about whether that can succeed, therefore no need for "try": it can always be increased unless it is already as high as memory-dynamic-max.
Perhaps "If the error occurs repeatedly, consider increasing the..."